salt/heroku/proxy_config.sls

```diff
@@ -1,4 +1,8 @@
-configure_heroku_proxy:
+{% for env in ['ci', 'rc', 'prod'] %}
+{% for app in ['bootcamp-ecommerce', 'micromasters', 'odl-open-discussions', 'odl-video'] %}
```

Comment:
One issue here is that odl-video only has a CI instance on Heroku... Should still work fine as long as we don't try running a state against it, but could lead to some confusion.